### ISSUE TITLE
Model Training

### ISSUE THREAD

**Issue by DesislavaNaydenova** [DesislavaNaydenova/pp5_cherry_leaves_mildew_detection#4]
**Description**  
Train a Convolutional Neural Network (CNN) model to classify healthy and mildew-infected leaves.  

**Tasks**  
- [x] Design the CNN architecture.
- [x] Train the model on the training set.
- [x] Evaluate the model on the validation set.
- [x] Save the trained model (`mildew_detector_model.h5`) to the outputs directory.
